lib/tasks/one_off_tasks/fix_location.rake

```diff
@@ -0,0 +1,31 @@
+namespace :fix_location do
+  desc "fix locations currently in the database"
+
+  task all_previous_records: :environment do
+    jobs_with_locations
+  end
+
+  def jobs_with_locations
+    stackoverflow_jobs = Job.where('url LIKE ?', '%stackoverflow%').all
+    stackoverflow_jobs.each do |job|
+      unless job.location =~ /(\w*, \w*)/
+        normalize_location(job)
+      end
+    end
+  end
+
+  def normalize_location(job)
+    if job.title =~ /(\w*, \w*)/
+      job.update_attribute(location: job.title[/(\w*, \w*)/])
```

Comment:
This seems like a repeat of how the location was originally generated. I'm not quite sure that you will eliminate situations where the title is for example: "Software engineer (JavaScript, Ruby, Go) at Company in (Cool Place, NY)"